tools/doc/html.js

```diff
@@ -390,3 +418,14 @@ function getId(text) {
   }
   return text;
 }
+
+const numberRe = /^(\d*)/;
+function versionSort(a, b) {
+  a = a.trim();
+  b = b.trim();
+  let i = 0;  // common prefix length
+  while (i < a.length && i < b.length && a[i] === b[i]) i++;
+  a = a.substr(i);
+  b = b.substr(i);
+  return +b.match(numberRe)[1] - +a.match(numberRe)[1];
```

Comment:
Oh, the capture is for zero or more digits (star, not plus.)  Right, that never fails to produce a match object.  Please disregard then.